receiver.go

```diff
@@ -81,7 +86,7 @@ func (r *Receiver) IssueCredit(credit uint32) error {
 //
 // NOTE: The behavior of drain is optional, as per the AMQP spec. Check with your individual
 // broker's documentation for implementation details.
-func (r *Receiver) DrainCredit(ctx context.Context) error {
+func (r *Receiver) DrainCredit(ctx context.Context, options *DrainCreditOptions) error {
```

Comment:
`_ *DrainCreditOptions` unless the linter doesn't complain.

Reply:
Updated anyways, it's a nice signal that we're not using, on purpose :)